apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -123,7 +123,7 @@ export class ModalPage {
       this.page.getByText(email),
       `Expected current email: ${email} to be visible on the notification screen`
     ).toBeVisible({
-      timeout: 10_000
+      timeout: 20_000
```

Comment:
Unfortunately I know we may really need to do this - but still wanted to ping if it may be unintentional after finding the issue in another place. We keep it?